cli/src/commands/content/channels.ts

```diff
@@ -18,6 +18,10 @@ export default class ChannelsCommand extends ContentDirectoryCommandBase {
           'ID': id.toString(),
           'Owner': JSON.stringify(c.owner.toJSON()),
           'Collaborators': c.collaborators.size,
+          'ChannelStateBloatBond': c.channelStateBloatBond.toString(),
```

Comment:
```suggestion
          'ChannelStateBloatBond': formatBalance(channelStateBloatBond),
```
It's more readable this way